src/store/storeConfig.ts

```diff
@@ -122,42 +117,36 @@ export const storeConfig = (kumaApi: KumaApi): StoreOptions<State> => {
         commit('SET_GLOBAL_LOADING', isLoading)
       },
 
-      async bootstrap({ dispatch, getters, state }) {
-        // check the Kuma status before we do anything else
-        await dispatch('config/getStatus')
-
-        // only dispatch these actions if the Kuma is online
-        if (getters['config/getStatus'] === 'OK') {
-          await Promise.all([
-            dispatch('fetchMeshList'),
-            dispatch('fetchDataplaneTotalCount'),
-            dispatch('config/bootstrapConfig'),
-          ])
-
-          // Validates if stored mesh exists and fetches the relevant sidebar data.
-          if (state.meshes.items.length > 0) {
-            const newStoredMesh = ClientStorage.get('selectedMesh')
-            let mesh: Mesh | undefined
-
-            // If a selected mesh is stored, check if it actually exists and use it only if it does.
-            if (newStoredMesh !== null) {
-              const existingMesh = state.meshes.items.find((mesh) => mesh.name === newStoredMesh)
-
-              if (existingMesh !== undefined) {
-                mesh = existingMesh
-              }
-            }
+      async bootstrap({ dispatch, state }) {
+        await Promise.all([
+          dispatch('fetchMeshList'),
+          dispatch('fetchDataplaneTotalCount'),
+          dispatch('config/bootstrapConfig'),
+        ])
+
+        // Validates if stored mesh exists and fetches the relevant sidebar data.
+        if (state.meshes.items.length > 0) {
+          const newStoredMesh = ClientStorage.get('selectedMesh')
+          let mesh: Mesh | undefined
+
+          // If a selected mesh is stored, check if it actually exists and use it only if it does.
+          if (newStoredMesh !== null) {
+            const existingMesh = state.meshes.items.find((mesh) => mesh.name === newStoredMesh)
 
-            if (mesh === undefined) {
-              // If the stored mesh doesn’t exist, use the first mesh instead.
-              mesh = state.meshes.items[0]
+            if (existingMesh !== undefined) {
+              mesh = existingMesh
             }
+          }
 
-            await dispatch('updateSelectedMesh', mesh.name)
-            await dispatch('sidebar/getInsights')
-          } else {
-            await dispatch('updateSelectedMesh', null)
+          if (mesh === undefined) {
+            // If the stored mesh doesn’t exist, use the first mesh instead.
+            mesh = state.meshes.items[0]
           }
+
+          await dispatch('updateSelectedMesh', mesh.name)
+          await dispatch('sidebar/getInsights')
+        } else {
+          await dispatch('updateSelectedMesh', null)
```

Comment:
All this green in this file is just removing the wrapping conditional checking to make sure that `status === 'OK'`